bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/factory/KNXHandlerFactory.java

```diff
@@ -126,4 +132,10 @@ private ThingUID getSerialBridgeThingUID(ThingTypeUID thingTypeUID, @Nullable Th
         String serialPort = (String) configuration.get(SERIAL_PORT);
         return new ThingUID(thingTypeUID, serialPort);
     }
+
+    public Collection<KNXBridgeBaseThingHandler> getBridges() {
+        return thingRegistry.getAll().stream().filter(thing -> thing.getHandler() instanceof KNXBridgeBaseThingHandler)
+                .map(thing -> (KNXBridgeBaseThingHandler) thing.getHandler())
+                .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
```suggestion
        return bridges.values();
```

Reply:
I see, you don't like iterating the thing registry ;-)
Ok, should be fine now.